skyplane/cli/cli.py

```diff
@@ -141,25 +141,29 @@ def cp(
         UsageClient.log_exception("cli_check_config", e, args, src_region_tag, dst_region_tag)
         return 1
 
-    if provider_src == "local" or provider_dst == "local":
+    if provider_src == ["local", "hdfs", "nfs"] or provider_dst == ["local", "hdfs", "nfs"]:
```

Comment:
I'm a little confused at the check for equality here. Will `provider_src` ever be of type `list`? Based on the context I see in the repo, `provider_src` and `provider_dst` will only ever be of type `str`.

Also, it will be slightly faster to check for provider_src in a set of "local", "hdfs", and "nfs":
```
>>> import timeit
>>> from statistics import median
>>> provider_src_def = "provider_src = 'nfs'"
>>> provider_src_list = "provider_src in ['local', 'hdfs', 'nfs']"
>>> median(timeit.repeat(stmt=provider_src_list, setup=provider_src_def, number=1000))
3.06189976981841e-05
>>> provider_src_set = "provider_src in {'local', 'hdfs', 'nfs'}"
>>> median(timeit.repeat(stmt=provider_src_set, setup=provider_src_def, number=1000))
1.744199835229665e-05
```

Reply:
Thanks for this, Kishan!